release.json

```diff
@@ -13,6 +13,20 @@
 		"JMXFETCH_VERSION": "0.33.1",
 		"JMXFETCH_HASH": "140dd8568b78f5a616da8664f34bb37397c3d3ed87af106d632714f7218e8e88"
 	},
+	"7.16.0-rc.7": {
+		"INTEGRATIONS_CORE_VERSION": "7.16.0-rc.7",
+		"OMNIBUS_SOFTWARE_VERSION": "7.16.0-rc.5",
```

Comment:
This went down from rc6 to rc5, is it intended?

Reply:
It is the same tag but I am fixing it.